src/main/kotlin/racing/ui/ResultView.kt

```diff
@@ -1,15 +1,26 @@
 package racing.ui
 
+import racing.domain.Cars
+
+private const val DISPLAY_MARK = "-"
+private const val DELIMITER = """:"""
+private const val DELIMITER_OF_WINNERS = ", "
+private const val MESSAGE_OF_WINNERS = "가 최종 우승했습니다."
+
 object ResultView {
 
     fun printResult() {
         println("실행 결과")
     }
 
-    fun print(positions: List<Int>) {
-        positions.forEach { position ->
-            println("-".repeat(position))
+    fun print(cars: Cars) {
+        cars.value().forEach { car ->
+            println("${car.name()} $DELIMITER ${DISPLAY_MARK.repeat(car.position)}")
         }
```

Comment:
`it`으로 대체할 수 있을 것 같아요. 🙂

```suggestion
        cars.value()
            .forEach { println("${it.name()} $DELIMITER ${DISPLAY_MARK.repeat(it.position)}") }
```